amazon-ec2-utils.spec

```diff
@@ -97,6 +97,9 @@ device mapping names.
 %{dracutlibdir}/modules.d/96ec2-utils/module-setup.sh
 
 %changelog
+* Wed Jan 17 2024 Christi Toa <toachris@amazon.com> - 2.3.1-2
+- Add support for --availability-zone-id to ec2-metadata
+
```

Comment:
Bump the minor and leave it at -1.

Also you'll want to make a branch that targets the upstream main and basically do the same bump there (except they're version 2.2.x).